### ISSUE TITLE
[Request] Edit e-mail templates

### ISSUE THREAD

**Issue by BarricadePL** [gogits/gogs#4808]
It would be nice if u can add e-mail templates editor for account activation and password reset, 
all kind of e-mails actually

**Comment by Fastidious** [comment]
You mean, [these](https://github.com/gogits/gogs/tree/master/templates/mail/auth)?

**Comment by BarricadePL** [comment]
Yes in Web editor for [these](https://github.com/gogits/gogs/tree/master/templates/mail/auth) templates.
I know u can edit them, but they are  lost when u upgrde.
I would like to edit them, and save in DB maybe.
They will ty always the same. No matter of files i download, because everyone got their own templates, and language. Or just add language support to the templates.
